### ISSUE TITLE
Fix request: Kotlin PMD detects issues in class, but not object

### ISSUE THREAD

**Issue by bhallrabo** [jborgers/PMD-jPinpoint-rules#607]
Hi,

When executing the PMD plugin check within IntelliJ we're finding that the rule violations aren't being detected in some of the class structures.

With the following structure it works fine:
Standard class structure
```
class DecimalFunctionClass {
    val df: DecimalFormat = DecimalFormat("#.###")

    fun format(value: Double) = df.format(value)
}
```

However in the following structure it doesn't find the violation:
Object class structure
```
object DecimalFunctionClass {
    val df: DecimalFormat = DecimalFormat("#.###")

    fun format(value: Double) = df.format(value)
}
```

We're listing this as a fix request as it can be something used by teams and might miss some of the violations
